api.js

```diff
@@ -213,7 +213,7 @@ server.use(
 
 // public files
 server.get(
-    { name: 'public_get', path: '/public/*' },
+    { name: 'public_get', path: '/public/*', exclude: true },
```

Comment:
Use a more descriptive key name than `exclude`. Currently it is hard to understand what is it for.

Reply:
Renamed to excludeRoute